Misc/NEWS.d/next/Library/2017-09-04-00-22-31.bpo-31334.9WYRfi.rst

```diff
@@ -0,0 +1,3 @@
+Fix ``poll.poll([timeout])`` in the ``select`` module for arbitrary negative
+timeouts on all OSes where it can only be a non-negative integer or -1.
```

Comment:
nitpick: "non-negative" is better known as "positive" :-)

Reply:
@haypo  "positive" doesn't include zero... :-)

Reply:
"strictly positive" doesn't include zero, "positive" does. Well, I don't really care. The current sentence is correct.

Reply:
"non-negative" means "positive or zero".